external/libfetch/http.c

```diff
@@ -1508,12 +1507,12 @@ http_get_proxy(struct url * url, const char *flags)
 static void
 http_print_html(FILE *out, FILE *in)
 {
-	size_t len;
-	char *line, *p, *q;
+	size_t len = 0;
+	char *line = NULL, *p, *q;
 	int comment, tag;
 
 	comment = tag = 0;
-	while ((line = fgetln(in, &len)) != NULL) {
+	while (getline(&line, &len, in) >= 0) {
```

Comment:
This matches what I have pushed upstream, but I forgot to bring that back into pkg, this will fix that ;)